src/cdomain/value/cdomains/unionDomain.ml

```diff
@@ -26,7 +26,10 @@ module Field = struct
     if equal f g then
       f
     else
-      raise Lattice.Uncomparable
+      match f, g with
+      | `Top, f
+      | f, `Top -> f
+      | _ -> raise Lattice.Uncomparable
```

Comment:
This essentially duplicates the included `FlatConf` now. Maybe we could just reuse that implementation and turn tops into exceptions?
```ocaml
match meet f g with
| `Top -> raise Lattice.Uncomparable
| m -> m
```

Reply:
It's `Bot` that would have to be turned into an exception here.

Reply:
Sorry, yes. But with that flipped, the logic should more closely describe the intent.